### ISSUE TITLE
Code highlighted in wrong place after saving a file

### ISSUE THREAD

**Issue by Ascii67** [sourcery-ai/sourcery#272]
### Checklist

- [x] I have searched the [Sourcery documentation](https://docs.sourcery.ai/) for the issue, and found nothing
- [x] I have checked there are no open [bugs](https://github.com/sourcery-ai/sourcery/issues?q=label%3Abug) referencing the same bug or problem

### Description
I added a custom rule in .sourcery.yaml which looks like this:
```
rules:
  - id: do-not-use-print-in-production-code
    description: Do not use print statements in production code.
    pattern: print
```
When I now start VSCode, the squiggly blue line is underneath the `print`statement. But when I now save the file, the squiggly blue line is right after the class definition:

![Bildschirmfoto vom 2022-09-04 19-49-42](https://user-images.githubusercontent.com/99130063/188326792-5a979daa-8796-4ca4-9866-cd92542e5469.png)

### Debug Information

IDE Version: 
VSCode 1.71.0

Sourcery Version: 
0.12.7

Operating system and Version:
Ubuntu 22.04

**Comment by reka** [comment]
Hi @Ascii67 ,

Thanks a lot for reporting this :bug: 
We're working on a fix.

I managed to reproduce this :bug: for classes that have a multiline docstring.
Did you observe it in other scenarios as well?

The 2 cases I've found so far:

Case A)
There's an empty line between the class definition and the docstring
=>
This empty line gets underlined.

```
class Example:

    """
    Example class to reproduce the bug. 

    https://github.com/sourcery-ai/sourcery/issues/272
    """

    def placeholder(self):
        print("anything")
```

Case B)
No empty line between the class definition and the docstring, but an empty line between the docstring and the 1st method
=>
The empty line between the docstring and the 1st method gets underlined.

```
class Example:
    """
    Example class to reproduce the bug. 

    https://github.com/sourcery-ai/sourcery/issues/272
    """

    def placeholder(self):
        print("anything")
```

**Comment by reka** [comment]
We're glad to hear that you're creating custom rules. :smiley: 
Do you have some feedback for us about them (besides this :bug:)?

For detecting calls to `print` in production code, another possibility is:

```
pattern: print(...)
```

This flags only calls to `print`, but doesn't flag other occurrences of `print`, e.g.:

```
alias_for_print = print
```

See also the [Flag `print` Statements recipe](https://docs.sourcery.ai/custom_rules/recipes/remove-debugging-statements/#flag-print-statements) in our docs.

**Comment by Ascii67** [comment]
Thanks for your answer, @reka!
> I managed to reproduce this bug for classes that have a multiline docstring. Did you observe it in other scenarios as well?

No, I just observed this in scenarios where I have a class with a multiline docstring with this structure: 
```py
class ClassName:

    """
    This is the docstring.
    """

    def __init__(self) -> None:
        ...

```

> Do you have some feedback for us about them (besides this bug)?

Since I've only just started with custom rules, I don't have any feedback yet but I didn't encounter any problems regarding custom rules and had a great experience with it!

**Comment by Hellebore** [comment]
Should be resolved with release of 0.12.8 - let us know if there are cases which aren't.
